src/Rinds/RindsSpecsDataSourcePoller.php

```diff
@@ -0,0 +1,72 @@
+<?php
+/**
+ * Handles polling and storage of RINDS specs
+ *
+ * @package WooCommerce Admin/Classes
+ */
+
+namespace Automattic\WooCommerce\Admin\Rinds;
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * RINDS specs data source poller class.
+ * This handles polling RINDS specs from JSON endpoints, and
+ * stores the specs in to the database as an option.
+ */
+class RindsSpecsDataSourcePoller {
+	const DATA_SOURCES = array(
+		'http://two.wordpress.test/rinds-specs.json',
+	);
+
+	/**
+	 * Polls the data sources for specs, persists those specs, and delete any
+	 * specs that no longer exist in the data sources.
+	 */
+	public static function poll_data_sources() {
```

Comment:
Since we still don't have a final URL for the source JSON data file, I kind of feel like we might want to make the polling of data sources configurable as a feature - so we can turn it off easily during the build flow.

Reply:
Makes sense, done